modules/setting-higher-pid-limit-on-machinepool.adoc

```diff
@@ -42,28 +42,28 @@ $ rosa create kubeletconfig -c my-cluster --name=set-high-pids --pod-pids-limit=
 +
 [source,terminal]
 ----
-$ rosa create machinepool -c <cluster_name> --kubelet-configs=<kubeletconfig_name> --name <machinepool_name>
+$ rosa create machinepool -c <cluster_name> --name <machinepool_name> --kubelet-configs=<kubeletconfig_name>
 ----
 ** For an existing machine pool:
 +
 [source,terminal]
 ----
-$ rosa edit machinepool -c <cluster_name> --kubelet-configs=<kubeletconfig_name> --name <machinepool_name>
+$ rosa edit machinepool -c <cluster_name> <machinepool_name> --kubelet-configs=<kubeletconfig_name>
```

Comment:
@laubai Same here, I'd suggest we make the `<machinepool_name>` the last part of the command